src/commands/api-mesh/log-get.js

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2021 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+const { Command } = require('@oclif/core');
+const logger = require('../../classes/logger');
+const { initSdk, initRequestId } = require('../../helpers');
+const { ignoreCacheFlag } = require('../../utils');
+const { getMeshId, getLogsByRayId } = require('../../lib/devConsole');
+require('dotenv').config();
+
+class FetchLogsCommand extends Command {
+	static args = [{ name: 'rayId', required: true, description: 'to fetch the log ' }];
+	static flags = {
+		ignoreCache: ignoreCacheFlag,
+	};
+
+	async run() {
+		await initRequestId();
+
+		logger.info(`RequestId: ${global.requestId}`);
+
+		const { args, flags } = await this.parse(FetchLogsCommand);
+
+		const ignoreCache = flags.ignoreCache;
+		const rayId = args.rayId;
+
+		const { imsOrgId, projectId, workspaceId } = await initSdk({
+			ignoreCache,
+		});
+
+		let meshId = null;
+		try {
+			meshId = await getMeshId(imsOrgId, projectId, workspaceId, meshId);
+			if (!meshId) {
+				throw new Error('MeshIdNotFound');
+			}
+		} catch (err) {
+			this.error(
+				`Unable to get mesh ID. Please check the details and try again. RequestId: ${global.requestId}`,
+			);
+		}
+
+		if (meshId) {
+			try {
+				const meshLog = await getLogsByRayId(imsOrgId, projectId, workspaceId, meshId, rayId);
+				if (meshLog) {
+					this.log('EventTimestampMs : %s', meshLog.eventTimestampMs);
+					this.log('Exceptions : %s', meshLog.exceptions);
+					this.log('Logs : %s', meshLog.logs);
+					this.log('Outcome : %s', meshLog.outcome);
```

Comment:
Can we use words instead of Pascalcase here

Reply:
yeah, sure will do